packages/legend-query-builder/src/stores/QueryBuilderResultState.ts

```diff
@@ -70,6 +71,10 @@ export class QueryBuilderResultState {
     );
   }
 
+  setCurrentExecutedQueryHashCode = (val: string | undefined): void => {
```

Comment:
Please do me a favor and convert `makeAutoObservable` to `makeObservable`. Also don't forget to add the fields accordingly

Reply:
Cleaned it up in `QueryBuilder` code. Do you want me to do it at other places as well?